broker/src/main/java/io/zeebe/broker/Broker.java

```diff
@@ -257,8 +258,7 @@ private AutoCloseable topologyManagerStep(
 
   private AutoCloseable metricsServerStep(
```

Comment:
Let's maybe rename this to monitoring server now that we do a bunch of things here?